modules/core/src/main/java/org/apache/ignite/internal/processors/odbc/jdbc/JdbcTableMeta.java

```diff
@@ -102,4 +116,9 @@ public String tableName() {
     @Override public int hashCode() {
         return Objects.hash(schemaName, tblName);
```

Comment:
Don't we consider `Objects.hash` bad practice?

Reply:
it's not related to changes for the ticket.